### ISSUE TITLE
Sponsor Pitch Deck for the Event

### ISSUE THREAD

**Issue by wolstaeb** [C3ETH/c3eth-innovate#7]
We need to develop a pitch deck for presenting to Kumpul, Emurgo, and IOG to explain the event and how it will unfold.

[Proposal](https://c3eth.github.io/catalyst/fund6/eth-proposals/connect-east-asian-entrepreneurs/) and [Challenge](https://c3eth.github.io/catalyst/fund6/challenges/proposer-outreach/campaign-brief/)

[Presentation Slide Deck](https://docs.google.com/presentation/d/1bO9b8H53_JrvvHrViJM40ySijxDI1bmEgeRnMjOYAOM/edit?usp=sharing) and [Pitch Deck](https://docs.google.com/document/d/1JrWU9DSH41qOvIiy_CjPh8AtgqfV_eouzqyZyjuXJ6Y/edit?usp=sharing)

### Structure

1. Introduction
2. Event 
3. Why
4. Who
5. How
6. Ask

### Event

- #2 
- #3 
- #4 
- #5 

### How

1. Week 1 - #3 
2. Week 2 - #3 
3. Week 3 - #4 
4. Week 4 - #4

**Comment by wolstaeb** [comment]
I've just set up a [Google Doc to work](https://docs.google.com/document/d/1JrWU9DSH41qOvIiy_CjPh8AtgqfV_eouzqyZyjuXJ6Y/edit?usp=sharing) and refered to it in the issue as **Related Notes**. We can work there instead of the pitch deck because making notes and playing with the content is easier in a document rather than a slide/presentation. We'll pull the summary material across to the slide deck when we're ready to start pulling it all together.

**Comment by YanTirta** [comment]
@wolstaeb , I've looked over the progress of [Pitch Deck](https://docs.google.com/document/d/1JrWU9DSH41qOvIiy_CjPh8AtgqfV_eouzqyZyjuXJ6Y/edit?usp=sharing) and it seems we are 99% there. How would we proceed now? Would we present this as is? Or would we make key bullet points of it when we meet kumpul.id and Emurgo today. Cause it may seem too wordy if we wanted to pitch it or you think that it's okay?

**Comment by YanTirta** [comment]
If we are planning to make [Related Docs](https://docs.google.com/document/d/1JrWU9DSH41qOvIiy_CjPh8AtgqfV_eouzqyZyjuXJ6Y/edit?usp=sharing) into a presentation that only has bullet points, how can I help you @wolstaeb ? Would any visual format I made yesterday be able to be used?

**Comment by wolstaeb** [comment]
Next steps should be to extract the key points from each segment of the text and turn those into bullet points for the slides. Then will will work on the slides title and how to present the bullets points visually. Either replacing them entirely with a visual cue or using visuals to argument the bullet points.

For example the Roadmap/Timeline is best presented visually with key dates highlighted.

**Comment by YanTirta** [comment]
Noted @wolstaeb , I'll be available in 1 hour 15 minutes. If there is a system where we can collaborate to work on the presentation visually I'm onboard. But if for the sake of efficiency it would not be possible, I can understand and will allocate my time to understand and memorize the pitch deck.

**Comment by YanTirta** [comment]
@wolstaeb , **Muzi from Entreprenerdy** has requested us to provide him with a date for our 1-on-1 session. **The slot he gave starts on the 15th of March to any day from 12pm UTC to 4pm UTC.** It's a first come first serve kind of thing.

**Comment by YanTirta** [comment]
**Yoram from Cardano 4 Climate Change has also set up a meeting with the people of https://www.goodwall.io/** , they are exploring collaboration potential with the people of Cardano Catalyst Asia. I reckon we could loop them in to Catalyst Innovate.

They are asking whether **Friday, 11th of March from 12.00 - 12.30 CET would be a good time?**

Would really appreciate your input @wolstaeb
